### PR TITLE
[calibration] bug fix indices

### PR DESCRIPTION
**Pull request by fabiencastan**
Fix alicevision/openMVG#260

### CODE REVIEW

File: src/openMVG/calibration/bestImages.cpp [hunk 1]
```diff
@@ -93,12 +92,12 @@ void selectBestImages(const std::vector<std::vector<cv::Point2f> >& imagePoints,
   precomputeCellIndexes(imagePoints, imageSize, calibGridSize, cellIndexesPerImage);
 
   // Init with 0, 1, 2, ...
-  remainingImagesIndexes.resize(validFrames.size());
+  remainingImagesIndexes.resize(imagePoints.size());
   for (std::size_t i = 0; i < remainingImagesIndexes.size(); ++i)
```
Comment (simogasp) [inline src/openMVG/calibration/bestImages.cpp]: `std::iota(remainingImagesIndexes.begin(), remainingImagesIndexes.end(), 0)`

File: src/openMVG/calibration/bestImages.cpp [hunk 2]
```diff
@@ -133,8 +132,12 @@ void selectBestImages(const std::vector<std::vector<cv::Point2f> >& imagePoints,
   }
   else
   {
-    OPENMVG_LOG_DEBUG("Info: Less valid frames (" << validFrames.size() << ") than specified maxCalibFrames (" << maxCalibFrames << ").");
-    bestImagesIndexes = validFrames;
+    OPENMVG_LOG_DEBUG("Info: Less valid frames (" << imagePoints.size() << ") than specified maxCalibFrames (" << maxCalibFrames << ").");
+    bestImagesIndexes.resize(imagePoints.size());
+    for(std::size_t i = 0; imagePoints.size(); ++i)
```
Comment (simogasp) [inline src/openMVG/calibration/bestImages.cpp]: `std::iota(bestImagesIndexes.begin(), bestImagesIndexes.end(), 0)`

**Comment by simogasp** [comment]
@fabiencastan Just wanted to see how `Request changes` works ;-)

File: src/openMVG/calibration/bestImages.cpp (hunk 2 above)
Comment (fabiencastan) [inline src/openMVG/calibration/bestImages.cpp]: Not sure why but `std::iota` is not available in `<numeric>` in my environment.
So I have not changed it.

Comment (simogasp) [inline src/openMVG/calibration/bestImages.cpp]: it's strange as there are lots of other places where it is used
https://github.com/alicevision/openMVG/search?utf8=%E2%9C%93&q=iota

File: src/software/calibration/main_rigCalibration.cpp [hunk 3]
```diff
@@ -415,6 +419,18 @@ int main(int argc, char** argv)
       return EXIT_FAILURE;
     }
 
+    double step = 1.0;
+    int nbFrames = feed.nbFrames();
```
Comment (simogasp) [inline src/software/calibration/main_rigCalibration.cpp]: const ?

Comment (fabiencastan) [inline src/software/calibration/main_rigCalibration.cpp]: done

**Pull request merged**